setup.cfg

```diff
@@ -35,3 +35,26 @@ requirements-doc =
     shuup
     Sphinx>=1.3,<1.6
     django-mptt==0.9.1
+
+requirements-dev =
+    -e .
+    black==20.8b1
+    isort==5.6.4
+    flake8==3.8.4
+    prequ
+
+requirements-tests =
+    -e .
+    codecov
+    cryptography==3.3.1  # Last known version that builds with Travis
+    beautifulsoup4==4.5.3
+    html5lib==0.999999999
```

Comment:
:DDD wtf is this versioning

Reply:
copied from tox, but that version has a really nice history behind it